src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -14,12 +14,188 @@
 #include "error_codes.h"
 #include "deps_format.h"
 
-pal::string_t fx_muxer_t::resolve_fx_dir(const pal::string_t& muxer_dir, runtime_config_t* runtime)
+pal::string_t resolve_impl_version_from_deps(const pal::string_t& deps_json)
 {
-    trace::verbose(_X("--- Resolving FX directory from muxer dir [%s]"), muxer_dir.c_str());
-    const auto fx_name = runtime->get_fx_name();
-    const auto fx_ver = runtime->get_fx_version();
-    const auto patch_roll_fwd = runtime->get_patch_roll_fwd();
+    trace::verbose(_X("--- Resolving %s version from deps json [%s]"), LIBHOSTPOLICY_NAME, deps_json.c_str());
```

Comment:
if pal::string is a wstring (on Windows), then the correct printf is %ls not %s (deps_json)